samples/drivers/can/counter/sample.yaml

```diff
@@ -4,6 +4,8 @@ tests:
   sample.drivers.can.counter:
     tags: can
     depends_on: can
+    integration_platforms:
+      - native_posix
```

Comment:
Could we add `native_posix_64` as well? I'd like to ensure 64-bit compatibility of the various CAN samples/test suites.

Reply:
we want to ensure 64 bit in many areas and this change does not remove that, this just limits coverage on PRs, but continues to run on everything else in different workflows.